webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -12,7 +12,7 @@
       <ds-button @click.stop.prevent="cropImage" class="crop-confirm" primary>
         {{ $t('contribution.teaserImage.cropperConfirm') }}
       </ds-button>
-      <ds-button @click="cancelCrop" class="crop-cancel" icon="close"></ds-button>
+      <ds-button @click.prevent="cancelCrop" class="crop-cancel" icon="close"></ds-button>
```

Comment:
Nice, can we add frontend tests for this behaviour? Or tests in general?

Reply:
Yes please.